fr.html

```diff
@@ -161,36 +161,36 @@
 <panel w=500 h=250>
 <pic src="pics/intro0.png" sx=400 sy=250></pic>
 <words x=155 y=20 w=90 no-bg>
-	Yeah.
+	Mouais.
 </words>
-<words x=196 y=65 w=270 no-bg>
-	Not only are common practices like lectures, cramming, and re-reading <i>boring</i>,
-	science has shown they <i>don't even work well</i>.*
+<words x=196 y=65 w=270 no-bg style="font-size:22px">
+	Non seulement les pratiques courantes comme les cours, le bachotage et la relecture <i>ennuyeuses</i>,
```

Comment:
Les pratiques courantes comme les cours, le bachotage et la relecture sont non seulement ennuyeuses, mais aussi d'une efficacité douteuse.

Reply:
I like your wording of the sentence, but it is too verbose to fit in the comic bubble. :(